components/advanced_text_editor/formatting_bar/formatting_bar.tsx

```diff
@@ -285,14 +277,17 @@ const FormattingBar = (props: FormattingBarProps): JSX.Element => {
                 </>
             )}
             <HiddenControlsContainer
-                ref={popperRef}
-                style={{...popper, zIndex: 2}}
-                {...attributes.popper}
+                ref={floating}
+                style={{
+                    position: strategy,
+                    top: y ?? 0,
+                    left: x ?? 0,
+                    zIndex: 20,
```

Comment:
using an object literal here will make `HiddenControlsContainer` re-render on every parent re-render.
Dunno though if it makes sense to `useMemo` the object or not.